### PR TITLE
feat: merge ping and uptime cogs

### LINKED ISSUE #721: Merge the ping and uptime cogs
## Description
Since both the ping and uptime commands are a similar 'category' of command it would make sense to merge both into one file/cog, this should be a simple change and there is no additional context.

## Reasoning
The reasoning is to make the code more concise and navigable.

## Would you like to implement this yourself?

- [x] I'd like to implement this feature myself
- [ ] Anyone can implement this feature

### PR DESCRIPTION
**Pull request by vcokltfre**
## Relevant Issues
closes #721 

## Description
Both the ping and uptime commands have been merged into a cog called "ping" which I believe is the most fitting name for it, though if there is a better name I am open to changing it. This has been tested locally and is working correctly, though of course anyone reviewing should see for themselves.

## Did you:

- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] Read all the comments in this template?
- [x] Ensure there is an issue open, or link relevant discord discussions?
- [x] Read the [contributing guidelines](https://pythondiscord.com/pages/contributing/contributing-guidelines/)?

### CODE REVIEW

File: bot/exts/evergreen/ping.py [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 
+from bot import start_time
 from bot.constants import Colours
 
 
 class Ping(commands.Cog):
-    """Ping the bot to see its latency and state."""
+    """Get info about the bot's ping and uptime."""
 
     def __init__(self, bot: commands.Bot):
```
Comment (Shivansh-007) [inline bot/exts/evergreen/ping.py:13]: In #718, all `commands.Bot` type hints were changed to `Bot` from `bot/bot.py`, we should change the type hint here and in the setup function.

Comment (ToxicKidz) [reply bot/exts/evergreen/ping.py:13]: I don't see why you'd do that, it's already done [here](https://github.com/python-discord/sir-lancebot/pull/718/files#diff-f325491078a141e8bdd8e16f8c395d06b4d11383e4b9d29876598fad7b1cc68a).

Comment (Shivansh-007) [reply bot/exts/evergreen/ping.py:13]: Alright, should be good then.

File: bot/exts/evergreen/ping.py [hunk 2]
```diff
@@ -21,7 +24,21 @@ async def ping(self, ctx: commands.Context) -> None:
 
         await ctx.send(embed=embed)
 
+    # Originally made in 70d2170a0a6594561d59c7d080c4280f1ebcd70b by lemon & gdude2002
+    @commands.command(name="uptime")
+    async def uptime(self, ctx: commands.Context) -> None:
+        """Responds with the uptime of the bot."""
```
Comment (decorator-factory) [inline bot/exts/evergreen/ping.py:30]: https://pythondiscord.com/pages/contributing/style-guide/#mood
> Imperative mood and present tense usage is preferenced when writing docstrings.
> Use:
> "Build an information embed."
> Don't use:
> "Returns an embed containing information."

**Review by Shivansh-007 — APPROVED** [review]

**Review by Xithrius — APPROVED** [review]

**Review by ToxicKidz — APPROVED** [review]
:shipit:

**Review by anand2312 — APPROVED** [review]
🚀

**Pull request merged**